data-serving/data-service/Dockerfile

```diff
@@ -38,6 +38,10 @@ USER node
 # Run with node env set to production.
 ENV NODE_ENV production
 
+# TODO: Remove this pending https://github.com/globaldothealth/list/issues/961.
+# Run with node debug mode set to http, net, and tls.
+ENV NODE_DEBUG http,net,tls
```

Comment:
you can set that up in the data.yaml and curator.yaml files instead of hardcoding those in the dockerfile

Reply:
Sure -- does this LGTU?

Reply:
perfect, thanks